src/components/QRShare/index.js

```diff
@@ -1,96 +1,91 @@
-import React, {Component} from 'react';
+import React, {forwardRef, useImperativeHandle, useRef, useState} from 'react';
 import {View} from 'react-native';
 import _ from 'underscore';
 import ExpensifyWordmark from '@assets/images/expensify-wordmark.svg';
 import QRCode from '@components/QRCode';
 import Text from '@components/Text';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
-import withTheme, {withThemePropTypes} from '@components/withTheme';
-import withThemeStyles, {withThemeStylesPropTypes} from '@components/withThemeStyles';
-import withWindowDimensions, {windowDimensionsPropTypes} from '@components/withWindowDimensions';
-import compose from '@libs/compose';
+import useTheme from '@styles/themes/useTheme';
+import useThemeStyles from '@styles/useThemeStyles';
 import variables from '@styles/variables';
 import {qrShareDefaultProps, qrSharePropTypes} from './propTypes';
 
-const propTypes = {
-    ...qrSharePropTypes,
-    ...windowDimensionsPropTypes,
-    ...withLocalizePropTypes,
-    ...withThemeStylesPropTypes,
-    ...withThemePropTypes,
-};
+function QRShare({innerRef, url, title, subtitle, logo, logoRatio, logoMarginRatio}) {
+    const styles = useThemeStyles();
+    const theme = useTheme();
 
-class QRShare extends Component {
-    constructor(props) {
-        super(props);
+    const [qrCodeSize, setQrCodeSize] = useState(1);
```

Comment:
Having small initial size for QR code made it change its size while page is opening. Ref: https://github.com/Expensify/App/issues/44436